### PR TITLE
Preparation for releasenotes 2023-05-10 and anouncement about WMTS in…

### PR DESCRIPTION
**Pull request by rebert**
… WGS84

### CODE REVIEW

**Comment by rebert** [comment]
@mariokeusen @consta-v @faselm @ltclm @marionb relnotes 20230510

**Review by mariokeusen — APPROVED** [review]
lgtm

File: chsdi/static/doc/source/releasenotes/index.rst [hunk 1]
```diff
@@ -15,6 +15,30 @@ Release Notes
...
+------------------------------------------
+
+API & applications
+******************
+
+`API <//api3.geo.admin.ch>`__
+'''''''''''''''''''''''''''''
+- Bug fixes
+- Announcements:
+    - The WMTS in `Plate-Carrée WGS1984 (EPSG:4326) <https://wmts.geo.admin.ch/EPSG/4326/1.0.0/WMTSCapabilities.xml>`__ is now in **lat/lon order**
```
Comment (ltshb) [inline chsdi/static/doc/source/releasenotes/index.rst:30]: `Plate-Carrée` ?

Comment (ltshb) [reply chsdi/static/doc/source/releasenotes/index.rst:30]: Should this announcements not be in a `WMTS <//wmts.geo.admin.ch>` section ?

**Review by ltshb — COMMENTED** [review]
Why not adding a release note to the date of the deployment of wmts on PROD ? From my understanding you will deploy WMTS with this fix before the next data deploy ? You could simply add a single release note announcement on the master branch and do a hotfix deploy for this.

**Pull request merged**